Libraries/LibLine/Editor.cpp

```diff
@@ -451,13 +451,13 @@ void Editor::initialize()
     for (auto& keybind : m_configuration.keybindings)
         register_key_input_callback(keybind);
 
-    Core::EventLoop::register_signal(SIGINT, [this](int) {
+    m_registered_signal_handlers.append(Core::EventLoop::register_signal(SIGINT, [this](int) {
         interrupted();
-    });
+    }));
 
-    Core::EventLoop::register_signal(SIGWINCH, [this](int) {
+    m_registered_signal_handlers.append(Core::EventLoop::register_signal(SIGWINCH, [this](int) {
         resized();
-    });
+    }));
```

Comment:
Maybe mention this change in the commit message? (if you didn't forget about cleaning it up after the patch that is :)

Reply:
Sorry I did not mean to commit this, let alone take credit for it...

Reply:
it's just a tiny patch, nothing to fight over

Reply:
No, it's a good catch because I didn't even notice ;) I had no intention to commit the other changes, I really don't want to ;)